.github/workflows/docsbuild-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: 'Release Version. Example: 11.1.0'
+        required: true
+      ansible-major-version:
+        description: 'Example 11'
+        required: true
```

Comment:
Let's derive this from the other input.

Reply:
Can you please suggest an example of getting major version from the input?

Reply:
Roughly, it'd be a step with something like
```yaml
- name: Retrieve the major version component
  id: compute-version
  run: |
    echo -n 'major-version=' >> "${GITHUB_OUTPUT}"
    echo '${{ inputs.ansible-version }}' | sed 's#\([0-9]\+\)\..*#\1#' >> "${GITHUB_OUTPUT}"
```
as the very first step, accessible through `${{ steps.compute-version.outputs.major-version }}` later on.